tests/core/test_neural_modules_pytorch.py

```diff
@@ -67,15 +70,7 @@ def test_constructor_TaylorNet(self):
 
     def test_call_TaylorNet(self):
         x_tg = nemo.core.neural_modules.NmTensor(
-            producer=None,
-            producer_args=None,
-            name=None,
-            ntype=nemo.core.neural_types.NeuralType(
-                {
-                    0: nemo.core.neural_types.AxisType(nemo.core.neural_types.BatchTag),
-                    1: nemo.core.neural_types.AxisType(nemo.core.neural_types.ChannelTag),
-                }
-            ),
+            producer=None, producer_args=None, name=None, ntype=NeuralType(ChannelType(), ('B', 'D'))
```

Comment:
could you add keyword axis=

Reply:
specifying kwarg's argument name is not required, here for brevity, I didn't use it